### DISCUSSION TITLE
querystring to Prisma Query generator?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [prisma/prisma#14312] [Q&A]
Are there any packages that does something like `https://github.com/ppetzold/nestjs-paginate` except it works on Prisma queries?

**Comment by TasinIshmam** [comment]
Hi @tyteen4a03  

Are you looking for something nestjs specific? In that case, I'm not sure if there is anything. 

However, there is the [prisma-relay-cursor-connection](https://github.com/devoxa/prisma-relay-cursor-connection) package that might be relevant. If you want to learn about integrating it in a NestJS proejct, it is covered in [this workshop about using NestJS and Prisma](https://youtu.be/mmbd5hcQUaY?t=4844).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
